Fhi.HelseId.Blazor/Fhi.HelseId.Blazor.csproj

```diff
@@ -5,7 +5,7 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <id>Fhi.HelseId.Blazor</id>
-    <Version>1.0.0-beta.2</Version>
+    <Version>1.0.0-alpha.1</Version>
```

Comment:
We are going from beta -> alpha here, is this intentional?

Reply:
Yes